control-plane-operator/controllers/hostedcontrolplane/ignitionserver/ignitionserver.go

```diff
@@ -188,6 +189,50 @@ func ReconcileIgnitionServer(ctx context.Context,
 	if hcp.Spec.Platform.Type != hyperv1.IBMCloudPlatform {
 		servingCertSecretName = manifests.IgnitionServerCertSecret("").Name
 	}
+
+	if len(hcp.Spec.ImageContentSources) > 0 {
```

Comment:
I don't think this is what we should look at. The hc/hcp .Spec.ImageContentSources specifies mirrors for the hosted cluster. Those mirrors may not be available to pods running in the control plane. We should be using any mirrors used by the management cluster. It also sounds like @bryan-cox may be addressing this use case in a different way.

Reply:
Yeah, after the conversation in slack, we agreed in use the `openshiftRegistryOverrides` to do the proper transformations.